core/src/test/java/com/linecorp/armeria/common/HttpResponseBuilderTest.java

```diff
@@ -261,6 +261,39 @@ void buildComplex() {
         assertThat(aggregatedRes.trailers().get("trailer-name")).isEqualTo("trailer-value");
     }
 
+    @Test
+    void buildWithHeadersAndPublisherContentAndTrailers() {
```

Comment:
https://github.com/line/armeria/blob/ff25d6f1e79c9fcfbbe632052a3102411554458f/core/src/main/java/com/linecorp/armeria/common/HttpResponseBuilder.java#L302-L303

I found that currently above code is not invoked by this `HttpResponsebuilderTest`! (`publisher + trailers` exist case)
So I added this unit test~!